README.md

```diff
@@ -150,31 +163,43 @@ Then open the network tab in web developer tools and reload the page. Inspect th
 from an API request and pull out the value of the `Authorization` header, for example
 `Token f1acac96cc79c4822e9010d23ab425231d580875`.
 
-Run `python scripts/load_incidents_v3.py --authz 'Token YOUR_AUTH_TOKEN' /path/to/directory_containing_incident_csvs`.
+Using the API token, run:
+```bash
+python scripts/load_incidents_v3.py --authz 'Token <YOUR_AUTH_TOKEN>' /path/to/directory_containing_incident_csvs
```

Comment:
Same idea here -- provide a fileshare reference for Azavea users; for non-Azavea users, describe the requirements and maybe provide sample CSV headers?